geo/src/main/java/org/apache/carbondata/geo/GeoHashUtils.java

```diff
@@ -83,6 +85,17 @@ public static long lonLat2GeoID(long longitude, long latitude, double oriLatitud
     return colRow2GeoID(ij[0], ij[1]);
   }
 
+  public static void validateUDFInputValue(Object input, String inputName, String datatype)
+      throws MalformedCarbonCommandException {
+    if (inputName.equalsIgnoreCase("gridSize") && (input == null
```

Comment:
can we declare "gridSize" as a constant and use it ?

Reply:
Done